code/mathematical_algorithms/src/tribonacci_numbers/tribonacci_numbers.js

```diff
@@ -0,0 +1,16 @@
+function tribonacci(n) {
+  if (n == 0){
```

Comment:
Add spaces around all operator, spaces before braces, and remove the last else statement. The very last line is missing a semicolon, which is somehow in the comment.